src/runtime-rs/crates/runtimes/src/static_resource.rs

```diff
@@ -128,40 +128,186 @@ fn get_sizing_info(annotation: Annotation) -> Result<(u64, i64, i64)> {
     // since we are *adding* our result to the config, a value of 0 will cause no change
     // and if the annotation is not assigned (but static resource management is), we will
     // log a *warning* to fill that with zero value
-    let period = annotation.get_sandbox_cpu_quota();
-    let quota = annotation.get_sandbox_cpu_period();
+    let period = annotation.get_sandbox_cpu_period();
+    let quota = annotation.get_sandbox_cpu_quota();
     let memory = annotation.get_sandbox_mem();
     Ok((period, quota, memory))
 }
 
 #[cfg(test)]
 mod tests {
     use super::*;
+    use kata_types::annotations::cri_containerd;
+    use std::collections::HashMap;
 
     #[test]
-    fn test_static_resource_mgmt() {
-        // data below should result in 2200 mCPU(round up to 3 vcpus) and 512 MiB of memory
-        let period: u64 = 100000;
-        let memory: i64 = 1048576 * 512; // 512 MiB
-        let quota: i64 = 220000;
-
-        let cpu = oci::LinuxCpu {
-            period: Some(period),
-            quota: Some(quota),
-            ..Default::default()
-        };
-        if let Ok(cpu_resource) = LinuxContainerCpuResources::try_from(&cpu) {
-            if let Some(v) = cpu_resource.get_vcpus() {
-                assert_eq!(v, 3);
-            }
+    fn test_static_resource_mgmt_sandbox() {
```

Comment:
These tests are what had you discovered the #5100 bug, I suppose.

They should go in different commits, as it's not directly related to #5100. This would allow a later git blame to correctly report what the code is about.

Reply:
Yes, I'd like to split them.